src/analysis/souffle/taint.dl

```diff
@@ -77,6 +85,10 @@
 // Rules
 //-----------------------------------------------------------------------------
 
+// A `tgt` is influenced by any `src` that is an operand to an operation where
+// the result is placed in `tgt`.
+influencedBy(owner, tgt, src) :- operationToOperands(owner, _, tgt, _, src).
```

Comment:
No, this is direct dependence, right?

Reply:
This is related to another conversation thread. Let us continue the discussion there.

Reply:
Removed `influencedBy`.